src/scripts/main.js

```diff
@@ -1,3 +1,136 @@
+let ascName = 1;
+let ascPosition = 1;
+let ascOffice = 1;
+let ascAge = 1;
+let ascSalary = 1;
+const body = document.querySelector('body');
+const form = document.createElement('form');
+const notification = document.createElement('div');
+const newRow = document.createElement('tr');
+
+body.appendChild(notification);
+body.appendChild(form);
+
+form.outerHTML = `<form class="new-employee-form">
+<label>Name: <input name="name" type="text" data-qa="name" ></label>
+<label>Position: <input name="position" type="text" data-qa="position" >
+</label>
+<label>Office: <select name="office" data-qa="office">
+<option value="Tokyo">Tokyo</option>
+<option value="Singapore">Singapore</option>
+<option value="London">London</option>
+<option value="New York">New York</option>
+<option value="Edinburgh">Edinburgh</option>
+<option value="San Francisco">San Francisco</option>
+</select>
+</label>
+<label>Age: <input name="age" type="number" data-qa="age" ></label>
+<label>Salary: <input name="salary" type="number"  data-qa="salary" ></label>
+<button name="button">Save to table</button>
+</form>`;
+
+const nameInput = document.querySelector('input[name="name"]');
+const positionInput = document.querySelector('input[name="position"]');
+const officeInput = document.querySelector('select[name="office"]');
+const ageInput = document.querySelector('input[name="age"]');
+const salaryInput = document.querySelector('input[name="salary"]');
+const button = document.querySelector('button');
+
+thead.addEventListener('click', e => {
+  const columnIndex = e.target.cellIndex;
+
+  switch (e.target.cellIndex) {
+    case 0:
+      sortStrings(ascName, columnIndex);
+      ascName *= -1;
+
+      break;
+    case 1:
+      sortStrings(ascPosition, columnIndex);
+      ascPosition *= -1;
+      break;
+
+    case 2:
+      sortStrings(ascOffice, columnIndex);
+      ascOffice *= -1;
+      break;
+
+    case 3:
+      rows.sort((a, b) => {
+        const current = a.cells[columnIndex].innerHTML;
+        const next = b.cells[columnIndex].innerHTML;
+
+        return ascAge * (Number(current) - Number(next));
+      });
+      ascAge *= -1;
+      break;
+
+    case 4:
+      rows.sort((a, b) => {
+        const current = a.cells[columnIndex].innerHTML;
+        const next = b.cells[columnIndex].innerHTML;
+
+        return ascSalary * (salaryToNumber(current) - salaryToNumber(next));
+      });
+      ascSalary *= -1;
+      break;
+  }
+
+  rows.forEach(x => tbody.appendChild(x));
+});
+
+tbody.addEventListener('click', e => {
+  rows.map(row => row.classList.remove('active'));
+  e.target.parentNode.classList.add('active');
+});
+
+button.addEventListener('click', (e) => {
+  e.preventDefault();
+  tbody.appendChild(newRow);
+  notification.setAttribute('data-qa', 'notification');
+
+  if (!nameInput.value || !ageInput.value || !officeInput.value
+    || !positionInput.value || !salaryInput.value) {
+    notification.classList.add('error');
```

Comment:
think you can also extract these logic to the separate function, like `notificationError(text)` you doing the same logic in three places, so will be easier to read to split into smaller ones:

```
function notificationError(text) {
  notification.classList.add('error');
  notification.textContent = text;
}
```